JASP-Engine/JASP-Engine.pro

```diff
@@ -88,8 +88,8 @@ exists(/app/lib/*) {
     InstallJASPRPackage.commands        = \"$$R_EXE\" CMD INSTALL --no-multiarch $$PWD/JASP
```

Comment:
Why didn't you also add --no-docs and --fake to the CMD INSTALL options for flatpak? Aren't they equally useful there?